frontend-react/src/utils/CustomRenderUtils.tsx

```diff
@@ -1,174 +1,92 @@
 import { FeatureFlagProvider } from "../contexts/FeatureFlagContext";
-
-import { mockToken } from "./TestUtils";
-
-/* Use this to generate fake useOktaAuth() hooks to pass into renderWithSession
- * This serves as our way of mocking different token, auth, and claims values */
-export const makeOktaHook = (_init?: Partial<IOktaContext>): OktaHook => {
-    const result = {
-        authState: {
-            accessToken: mockToken(),
-            ..._init?.authState,
-        },
-        oktaAuth: {
-            ..._init?.oktaAuth,
-        } as OktaAuth,
-    };
-    return () => result;
-};
-
-/*
-    To create a custom renderer, you must create a functional
-    component and a custom render function.
-
-    @see: https://testing-library.com/docs/react-testing-library/setup/#custom-render
-*/
-
-/*
-    Use `renderWithRouter()` from this module as our standard
-    renderer across most tests. This will prevent hitting the
-    React error when rendering for unit tests.
-*/
-const RouterWrapper: FC = ({ children }) => {
-    return (
-        <BaseWrapper>
-            <BrowserRouter>{children}</BrowserRouter>
-        </BaseWrapper>
+import { appRoutes } from "../AppRouter";
+
+interface AppWrapperOptions {
+    initialRouteEntries?: string[];
+    restHookFixtures?: Fixture[];
+}
+
+interface TestRouterProps {
+    children: React.ReactNode;
+    initialEntries?: string[];
+}
+
+/**
+ * Dynamically makes the supplied children the return element for all
+ * routes.
+ * FUTURE_TODO: Remove this once okta user/session mocking is easier
+ * and use <AppRouter /> instead.
+ */
+const TestRoutes = ({ children }: TestRouterProps) => {
+    const routes = useRoutes(
+        appRoutes.map((r) => ({ ...r, element: children }))
     );
-};
 
-export const SessionWrapper = ({ children }: PropsWithChildren<{}>) => {
-    return (
-        <BaseWrapper>
-            <RouterWrapper>
-                <SessionProvider>{children}</SessionProvider>
-            </RouterWrapper>
-        </BaseWrapper>
-    );
+    return routes;
 };
 
-export const QueryWrapper =
-    (client: QueryClient = getTestQueryClient()) =>
-    ({ children }: PropsWithChildren<{}>) =>
-        (
-            <BaseWrapper>
-                <QueryClientProvider client={client}>
-                    <AuthorizedFetchProvider initializedOverride={true}>
-                        {children}
-                    </AuthorizedFetchProvider>
-                </QueryClientProvider>
-            </BaseWrapper>
+export const AppWrapper = ({
+    initialRouteEntries,
+    restHookFixtures,
+}: AppWrapperOptions = {}) => {
+    // FUTURE_TODO: Replace children with <AppRouter /> if initialRouteEntries after mocking okta users
+    // in tests is made easier for better coverage as we'd be able to test through
+    // any custom route wrappers.
+    // FUTURE_TODO: Remove MockResolver and restHookFixtures when removing react-hooks.
+    return ({ children }: PropsWithChildren<{}>) => {
```

Comment:
Minor point, and I know this existed beforehand, but I think we can just define `children` explicitly since it's clearer with props and `PropsWithChildren` doesn't give us anything more than that.